ext/RastersArchGDALExt/cellarea.jl

```diff
@@ -0,0 +1,98 @@
+function _great_circle_bearing(lon1::AbstractFloat, lat1::AbstractFloat, lon2::AbstractFloat, lat2::AbstractFloat)
+    dLong = lon1 - lon2
+
+    s = cos(lat2)*sin(dLong)
+    c = cos(lat1)*sin(lat2) - sin(lat1)*cos(lat2)*cos(dLong)
```

Comment:
These could all be cosd or sind, so we might not need deg2rad in the invoking method.

Reply:
I didn't know about those, thanks!